terraform.tf.j2

```diff
@@ -108,3 +108,33 @@ locals {
   }
 }
 
+locals {
+
+  env_prefix = {
+    development = "dev."
+    qa          = "qa."
+    stage       = "stg."
+    integration = "int."
+    preprod     = "pre."
+    production  = ""
+  }
+
+}
+
+locals {
+
+  dataworks_domain_name = "dataworks.dwp.gov.uk"
```

Comment:
Sorry, I meant for this to be `{{dataworks_domain_name}}` - this will then automatically get populated into `terraform.tf` during a `make bootstrap` run.

Reply:
Matt, have done the changes. Can you please check now.